src/v1/controller/login/github/Callback.ts

```diff
@@ -138,6 +144,10 @@ export const callbackSchemaType: FastifySchema<CallbackRequest> = {
                 type: "string",
                 format: "uuid-v4",
             },
+            platform: {
+                type: "string",
+                nullable: true,
+            },
```

Comment:
```suggestion
            needArouse: {
                type: "boolean",
                nullable: true,
            },
```

Reply:
I think both choice are ok. But if we need to pick one I think `platform` is more decoupled with business logic comparing to `needArouse` (or `needLaunchApp`) which is an imperative action.

Reply:
emmmm ok